plugin/validator/handler.go

```diff
@@ -20,12 +20,16 @@ import (
 	"net/http"
 
 	"github.com/drone/drone-go/drone"
-
 	"github.com/drone/drone-go/plugin/logger"
 
 	"github.com/99designs/httpsignatures-go"
 )
 
+const (
+	statusSkip  = 498
```

Comment:
this is a minor nit, should it explicitly mention that the variable is associated with http statuses.

eg httpStatusSkip ?